recipes/nextflow/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: nextflow
+  version: "0.19.3"
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: nextflow
+  url: https://github.com/nextflow-io/nextflow/releases/download/v0.19.3/nextflow
+  md5: 7db3f250648bb818853664dd9a270556
+
+requirements:
+  run:
+    - java-jdk >=7.0
+
+test:
+  commands:
+  - nextflow -v
```

Comment:
The indentation could be fixed here, otherwise it looks good to me.
Are you sure it does not work under OSX?